tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -58,7 +58,7 @@ private static IPublicClientApplication CreatePca()
                             .Create(s_clientIdForPublicApp)
                             .WithAuthority(GetAuthority())
                             .WithLogging(Log, LogLevel.Verbose, true)
-                            .WithRedirectUri(DefaultOsBrowserWebUi.FindFreeLocalhostRedirectUri()) // required for DefaultOsBrowser
+                            .WithRedirectUri("https://localhost:12346") // required for DefaultOsBrowser
```

Comment:
>WithRedirectUri [](start = 29, length = 15)

Would it make sense to have something like WithLocalhostRedirectUri? which takes no params, but does the job of selecting the free port?

Reply:
Yes, just a bit worried that we're adding too many methods to the API. In any case, can be added later.